tests/__snapshots__/emotion.js.snap

```diff
@@ -31,17 +31,7 @@ exports[`Box renders with props 1`] = `
 @media screen and (min-width:40em) {
   .emotion-0 {
     margin: 8px;
-  }
-}
-
-@media screen and (min-width:40em) {
```

Comment:
I think I know what this change is about – this is because the previous version of styled-system used in this snapshot ended up with Emotion generating multiple `@media` rules, but the styles should be the same. The extra styles being added in the other snapshot are from the duplicate `flexbox` on the Flex component